File: LLM-Translations/GPT-4/codenet/Go/C/s716011659.c

#include <stdio.h>
#include <math.h>
#include <stdlib.h>

#define INF 10000000000.0

int main() {
    int n, i;
    scanf("%d", &n);

    int arrA[n];
    for (i = 0; i < n; i++) {
        scanf("%d", &arrA[i]);
    }

    int cumSumA[n + 1];
    cumSumA[0] = 0;
    for (i = 0; i < n; i++) {
        cumSumA[i + 1] = cumSumA[i] + arrA[i];
    }

    double ans = INF;
    for (i = 1; i < n; i++) {
        if (fabs((double)(cumSumA[n] - (cumSumA[i] * 2))) < ans) {
            ans = fabs((double)(cumSumA[n] - cumSumA[i] * 2));
        }
    }

    printf("%d\n", (int)ans);

    return 0;
}

File: LLM-Translations/GPT-4/codenet/Go/Python/s716011659.py
import sys
from typing import List

def main():
    n = int(input().strip())
    arrA = list(map(int, input().strip().split()))

    cumSumA = [0] * (n + 1)
    cumSumA[0] = 0
    for i, a in enumerate(arrA):
        cumSumA[i + 1] = cumSumA[i] + a

    ans = float('inf')
    for i in range(1, n):
        ans = min(ans, abs((cumSumA[n] - cumSumA[i] * 2)))

    print(int(ans))

if __name__ == "__main__":
    main()


